container-images/tcib/base/os/tobiko/run_tobiko.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/sh
 
+set -x
+
 TOBIKO_DIR=/var/lib/tobiko
 
 # assert mandatory variables have been set
```

Comment:
Not related to the PR. Maybe we can put quotation marks around the ${TOBIKO_TESTENV} here. For example, when the user wants to run a single test the TOBIKO_TESTENV variable will contain:
```
neutron -- tobiko/tests/scenario/neutron/test_qos.py::QoSNetworkTest::test_ping_dscp
```
and this will lead to:
```
tortuga.txt: line 4: [: syntax error: `--' unexpected
```

Reply:
good catch, thanks!